crates/bitwarden-threading/src/cancellation_token.rs

```diff
@@ -0,0 +1,66 @@
+pub use tokio_util::sync::CancellationToken;
+
+#[cfg(target_arch = "wasm32")]
+pub mod wasm {
```

Comment:
If the whole file is wasm-only, we should probably just put the cfg in the lib.rs, and then we avoid one nesting level here:
```
#[cfg(target_arch = "wasm32")]
pub mod cancellation_token;
```

Reply:
It's not, I'm re-exporting the cancellation token as a convenience regardless of wasm.